### DISCUSSION TITLE
Set Pod Environment Variable

### DISCUSSION THREAD

**Discussion by dspeck1** [cloudnative-pg/cloudnative-pg#785] [Q&A]
We are trying to set a pod's environment variable for the database pods in the cluster.  Pod Template Spec seems to be the place, but do not know how to configure the Cluster specification.  Where does this go or does this work (based on the linked issues)?  

https://github.com/cloudnative-pg/cloudnative-pg/blob/main/docs/src/api_reference.md#podtemplatespec

**Comment by gbartolini** [comment]
We deliberately removed pod templates from the v1 API. We might consider adding them in v2 (currently we are not working on that though).

The only available option at the moment is to add it in the container image.

**Reply by netthier** [reply]
I have multiple clusters running behind a proxy, and I'd need to set HTTPS_PROXY and HTTP_PROXY variables to be able to back them up.
If environment variables are unplanned for now, is there a better way of doing this than hardcoding those variables in the container?
